File: js/URI/1010.js
var input = require('fs').readFileSync("URI/in", 'utf-8');
var lines = input.split('\n');
var teste = lines.shift();
var teste2 = teste.split(' ')
var cod1 = parseInt(teste2.shift());
var qt1 = parseInt(teste2.shift());
var val1 = parseFloat(teste2.shift());
teste = lines.shift();
teste2 = teste.split(' ')
var cod2 = parseInt(teste2.shift());
var qt2 = parseInt(teste2.shift());
var val2 = parseFloat(teste2.shift());
var tot = qt1 * val1 + qt2 * val2;
tot = tot.toFixed(2);
console.log(`VALOR A PAGAR: R$ ${tot}`);

File: c#/uri/1010.cs
using System;
using System.Linq;
class URI{
    static void Main(string[] args){
        float a, b, c, total = 0;
        for(int i = 0; i < 2; i++){
            var input = Console.ReadLine().Split(' ').Select(s => float.Parse(s)).ToArray();
            a = input[0]; b = input[1]; c = input[2];
            total+=b * c;
        }
        Console.WriteLine("VALOR A PAGAR: R$ {0:f2}", total);
    }
}